.github/workflows/process-results.yml

```
@@ -32,15 +36,12 @@ jobs:
         uses: 'google-github-actions/setup-gcloud@v1'
         with:
           version: '>= 363.0.0'
-      - uses: actions/download-artifact@v3
-        with:
-          name: latest-result
-          path: ./
       - name: Upload results to GCS
         run: |
           FILENAME=$(date +%Y-%m-%d)
-          cp latest.$EXTENSION $FILENAME.$EXTENSION
-          gzip $FILENAME.$EXTENSION
+          cp artifacts/latest.$EXTENSION artifacts/$FILENAME.$EXTENSION
+          gzip artifacts/$FILENAME.$EXTENSION
 
-          gsutil -m cp -r $FILENAME.$EXTENSION.gz gs://egq-runziggurat-$REPOSITORY-bucket/results/$NAME
+          ls -a
+          gsutil -m cp -r artifacts/.  gs://egq-runziggurat-$REPOSITORY-bucket/results/$NAME
```

I think we need to use a wildcard here:
```suggestion
          gsutil -m cp -r artifacts/*  gs://egq-runziggurat-$REPOSITORY-bucket/results/$NAME
```